vnext/Desktop/React.Windows.Desktop.vcxproj

```
@@ -116,7 +116,7 @@
         $(ReactNativeDir)\ReactCommon\react\renderer\graphics\platform\cxx;
         %(AdditionalIncludeDirectories);
       </AdditionalIncludeDirectories>
-      <AdditionalOptions>%(AdditionalOptions) /Zc:strictStrings /bigobj</AdditionalOptions>
+      <AdditionalOptions>/await %(AdditionalOptions) /Zc:strictStrings /bigobj</AdditionalOptions>
```

nit: but put everything either before or after the existing additional options 
```suggestion
      <AdditionalOptions>%(AdditionalOptions) /await /Zc:strictStrings /bigobj</AdditionalOptions>
```